### PR TITLE
Doc upgrade

### PR DESCRIPTION
**Pull request by michaelbenayoun**
As per title.

### CODE REVIEW

**Comment by philschmid** [comment]
the [[autodoc]] seems not to work see CI error

File: docs/source/package_reference/export.mdx [hunk 1]
```diff
@@ -16,8 +16,75 @@ limitations under the License.
...
+Besides, you can check if the exported model is valid via [`~optimum.exporters.neuron.convert.validate_model_outputs`], which compares
+the compiled model on Neuron devices's output to the PyTorch model on CPU's output.
+
+[[autodoc]] exporters.convert.export_neuron
+
+[[autodoc]] exporters.convert.export_neuronx
+
+[[autodoc]] exporters.convert.export
+
+[[autodoc]] exporters.convert.validate_model_outputs
```
Comment (JingyaHuang) [inline docs/source/package_reference/export.mdx:37]: As I mentioned, it's not going to be accepted by doc-builder. But I do agree that we shall enable that.

File: docs/source/package_reference/export.mdx [hunk 2]
```diff
@@ -16,8 +16,75 @@ limitations under the License.
...
+| MPNet                  | feature-extraction, fill-mask, multiple-choice, question-answering, text-classification, token-classification                                 |
+| RoBERTa                | feature-extraction, fill-mask, multiple-choice, question-answering, text-classification, token-classification                                 |
+| RoFormer               | feature-extraction, fill-mask, multiple-choice, question-answering, text-classification, token-classification                                 |
+| XLM                    | feature-extraction, fill-mask, multiple-choice, question-answering, text-classification, token-classification                                 |
+| XLM-RoBERTa            | feature-extraction, fill-mask, multiple-choice, question-answering, text-classification, token-classification                                 |
+| Stable Diffusion       | N/A                                                                                                                                           |
```
Comment (JingyaHuang) [inline docs/source/package_reference/export.mdx:81]: text-to-image ?

**Review by JingyaHuang — COMMENTED** [review]
Left some nits, thanks for working on it!!

**Pull request merged**